packages/api-page-builder/src/graphql/crud/system.crud.ts

```diff
@@ -25,8 +25,8 @@ export interface CreateSystemCrudParams {
 }
 export const createSystemCrud = (params: CreateSystemCrudParams): SystemCrud => {
     const { context, storageOperations, getTenantId } = params;
-    const onBeforeInstall = createTopic<OnBeforeInstallTopicParams>();
-    const onAfterInstall = createTopic<OnAfterInstallTopicParams>();
+    const onBeforeInstall = createTopic<OnBeforeInstallTopicParams>("pageBuilder.onBeforeInstall");
```

Comment:
If we're following `on{conjunction}{noun}{subordinatenoun}{action}`, then we're missing a noun here.

Can I suggest `onBeforeSystemInstall` ?